powerline/matchers/vim/plugin/ctrlp.py

```diff
@@ -0,0 +1,54 @@
+# vim:fileencoding=utf-8
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+try:
+    import vim
+except ImportError:
+    vim = object()
+
+import os
```

Comment:
Though no, when initialization function is moved importing vim is no longer needed.